PIL/Image.py

```diff
@@ -1122,6 +1186,7 @@ def filter(self, filter):
         return merge(self.mode, ims)
 
     def getbands(self):
+        # type: () -> Tuple[Any]
```

Comment:
**update*: so, probably `Tuple[Mode, ...]`, which is the same as you specified, except that Mode is (in an updated commit I made) an alias for Text, not str.